s_vae/training.py

```diff
@@ -57,9 +57,9 @@ def train(config: dict):
     trainer.fit(model=engine, train_dataloader=train_loader, val_dataloaders=valid_loader)
     trainer.test(test_dataloaders=valid_loader)
 
-    engine.set_fix_var(False)
-    engine.freeze_model()
-    engine.freeze_var(False)
+    #engine.set_fix_var(False)
```

Comment:
Maybe this can be a problem. So basically I did not know how to turn this off completely for the Bernoulli decoder.

Reply:
Just add an if function for Bernulli